src/frontends/paddle/src/op/set_value.cpp

```diff
@@ -0,0 +1,120 @@
+// // Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+#define MAX_VALUE(T) std::numeric_limits<T>::max()
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+
+std::shared_ptr<default_opset::Constant> get_max_value_by_dtype(ov::element::Type dtype) {
+    if (dtype == element::f32)
+        return default_opset::Constant::create(dtype, {}, {MAX_VALUE(float)});
+    else
+        return default_opset::Constant::create(dtype, {}, {MAX_VALUE(int)});
+};
+
+Output<Node> handle_minus_index(const OutputVector& node, const Output<Node>& dim) {
+    const auto zero = default_opset::Constant::create(node[0].get_element_type(), {1}, {0});
+    const auto concat_node = std::make_shared<default_opset::Concat>(node, 0);
+    const auto mask = std::make_shared<default_opset::Less>(concat_node, zero);
+    const auto res = std::make_shared<default_opset::Add>(concat_node, dim);
+    return std::make_shared<default_opset::Select>(mask, res, concat_node);
+}
+
+NamedOutputs set_value(const NodeContext& node) {
+    const auto input_node = node.get_input("Input");
+    auto value_node = node.get_input("ValueTensor");
+    const auto input_shape = input_node.get_partial_shape().get_shape();
```

Comment:
If use the get_shape(), that means that the shape is static. Can use the 'ShapeOf ' to support dynamic shape

Reply:
done